cmd/flowrunner/testdata/flows/dynamic_groups_test.json

```diff
@@ -40,43 +67,74 @@
                     "type": "msg_created"
                 },
                 {
-                    "created_on": "2018-07-06T12:30:08.123456789Z",
+                    "created_on": "2018-07-06T12:30:10.123456789Z",
                     "step_uuid": "692926ea-09d6-4942-bd38-d266ec8d3716",
                     "type": "contact_urn_added",
                     "urn": "tel:+250781234567"
                 },
                 {
-                    "created_on": "2018-07-06T12:30:09.123456789Z",
+                    "created_on": "2018-07-06T12:30:11.123456789Z",
+                    "groups_added": [
+                        {
+                            "name": "MTN Callers",
+                            "uuid": "db60d4dd-b4eb-46bf-9cdb-65d3d0e75bba"
+                        }
+                    ],
+                    "step_uuid": "692926ea-09d6-4942-bd38-d266ec8d3716",
+                    "type": "contact_groups_changed"
+                },
+                {
+                    "created_on": "2018-07-06T12:30:12.123456789Z",
                     "msg": {
                         "channel": {
                             "name": "Android Channel",
                             "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
                         },
-                        "text": "Current groups: [\"Males\",\"Old Men\"]",
+                        "text": "Current groups: [\"Males\",\"Old Men\",\"MTN Callers\"]",
```

Comment:
contact URN additions weren't previously triggering a dynamic group evaluation.. despite me apparently including one in this test